openreview/openreview.py

```diff
@@ -8,16 +8,19 @@
 import datetime
 import builtins
 
-def epoch_time():
-    return int((datetime.datetime.utcnow() - datetime.datetime(1970, 1, 1)).total_seconds()*1000)
-
 class OpenReviewException(Exception):
     pass
 
 class Client(object):
 
-    def __init__(self, baseurl = None, process_dir = '../process/', webfield_dir = '../webfield/', username = None, password = None):
-        """CONSTRUCTOR DOCSTRING"""
+    def __init__(self, baseurl = None, username = None, password = None):
+        """
```

Comment:
Just a note: I do not think this doc-string will show up in the documentation.

Reply:
Do you know why?

I copied the example from Elastic Search:

https://github.com/elastic/elasticsearch-py/blob/master/elasticsearch/client/__init__.py#L174

Reply:
Sorry, looks like it does show up. I made an incorrect assumption that any function with a leading underscore does not show up in the documentation.